### ISSUE TITLE
Cant Import Captions

### ISSUE THREAD

**Issue by ClipSkipper** [cocktailpeanut/fluxgym#225]
Anytime importing captions, the script will execute training on the first image and only the first image with the set number of steps then exit.  The presence of a matching .txt file to the image seems to prevent it from iterating onto the next image or even tallying multiple images as part of the dataset.

**Comment by HKCLynn** [comment]
I have the same problem with you. I don't know whether the way I import captions is wrong.

**Comment by ClipSkipper** [comment]
@HKCLynn Did you ever find a solution for this?, manualy copy pasting them is not great.

**Comment by HKCLynn** [comment]
The command line in terminal runs normally, so I gave up using the UI interface.

**Comment by ClipSkipper** [comment]
good shout, didnt even think to try.

**Comment by ostap667inbox** [comment]
Yeah, I have the same problem. This bug is still not fixed. When I open a dataset with images and txt files, I see that the captions are substituted correctly. However, the total number of images is doubled because fluxgym incorrectly counts txt files as images as well.

**Comment by Latentnaut** [comment]
I have followed the instruction in Github page but there are not captions imported.
